runtime/themes/monokai_pro_spectrum.toml

```diff
@@ -87,7 +87,10 @@
 "diff.minus" = "red"
 
 # make diagnostic underlined, to distinguish with selection text.
-diagnostic = { modifiers = ["underlined"] }
+"diagnostic.warning" = { underline = { color = "orange", style = "curl" } }
+"diagnostic.error" = { underline = { color = "magenta", style = "curl" } } # maybe should be red?
```

Comment:
@blt-r `"magenta"` is not defined as a color in this theme, and causes problems. (See https://github.com/helix-editor/helix/issues/5249)

`"red"` must be used instead. (See `[palette]` a few lines down.)

(`"error"` using `"magenta"` is broken, too.)

Reply:
Oh, I didn't notice that magenta is not defined, just used whatever `"error"` used

Reply:
No problem. It's fixed now.